core/templates/components/question-directives/modal-templates/change-question-difficulty-modal.controller.ts

```diff
@@ -30,11 +30,7 @@ angular.module('oppia').controller('ChangeQuestionDifficultyModalController', [
       $uibModalInstance: $uibModalInstance
     });
 
-    var init = function() {
-      $scope.linkedSkillsWithDifficulty =
-        linkedSkillsWithDifficulty;
-      $scope.skillIdToRubricsObject = skillIdToRubricsObject;
-    };
-    init();
+    $scope.linkedSkillsWithDifficulty = linkedSkillsWithDifficulty;
```

Comment:
Why this change?

Reply:
Also, should we be using $onInit()?

Reply:
I believe `$onInit` is only used with `ctrl`. I've changed this controller because it was creating a function and then executing it to set the properties. Instead, I just set the properties directly, without a function.

Reply:
I see. OK thanks for the explanation!